docs/cookbook.rst

```diff
@@ -19,3 +32,37 @@ wherever you need to do them::
     from screenpy_adapter_allure import AllureAdapter
 
     the_narrator.attach_adapter(AllureAdapter())
+
+After that the steps are attached to the allure report
+
+.. image:: ./allure_with_plugin.png
+  :width: 768
+  :alt: Screenshot without plugin has not the steps
+
+Attach Screenshot from selenium
+===============================
+
+If you are using allure in a selenium project using `ScreenpPy Selenium <https://screenpy-selenium-docs.readthedocs.io/en/latest/>`__
+
+You can attach a new screenshot to the report using the action `SaveScreenshot <https://screenpy-selenium-docs.readthedocs.io/en/latest/extended_api/actions.html#screenpy_selenium.actions.SaveScreenshot>`__
+
+For example you can attach an screenshot as PNG::
+
+    from screenpy import Actor
+    from allure_commons.types import AttachmentType
+
+    filepath = 'screenshot.png'
+
+    the_actor = Actor.named("Perry").who_can(BrowseTheWeb.using_firefox())
+
+    the_actor.attempts_to(
+        SaveScreenshot.as_(filepath).and_attach_it_with(
+            attachment_type=AttachmentTypes.PNG,
+        ),
+    )
+
+In the github example we go to the github page and find the `screenpy_examples` repository
```

Comment:
I think you could link the github example here:
```rst
In the `github example <https://github.com/ScreenPyHQ/screenpy_examples/blob/trunk/screenpy_selenium/github/features/test_github_search.py#L44-L54>`__,
we go to the github page
and find the ``screenpy_examples`` repository
```

(note the double-`s around screenpy_examples—RST uses backticks as special characters, so you need to double them if you want it to look like code instead.)